dnstable/query.c

```diff
@@ -745,6 +745,7 @@ query_iter_next_name_indirect(void *clos, struct dnstable_entry **ent, uint8_t t
 				case WDNS_TYPE_PTR:
 				case WDNS_TYPE_MX:
 				case WDNS_TYPE_SRV:
+				case WDNS_TYPE_NSEC:
```

Comment:
For NSEC rdata, `add_rrtype_to_key` will conflict with the bitmap portion. Like with SOA records, the NSEC rrtype should be left out of this switch and the similar switch below.